kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -243,6 +251,11 @@ public boolean importChildRecord(Map<String, Object> record) {
             return false;
         }
 
+        //validate if it's an updated record compared to one from database
+        if (child.getUpdatedDateNic() != null && (mctsUpdatedDateNic == null || child.getUpdatedDateNic().isAfter(mctsUpdatedDateNic))) {
+            return true;
+        }
```

Comment:
same as the mother comment. if the update is being dropped, don't you want to return false here?